docker/bookkeeper/Dockerfile

```diff
@@ -16,15 +16,17 @@ RUN apt-get update \
 &&  mkdir -p /opt \
 &&  cd /opt
 
-RUN wget http://www.apache.org/dist/bookkeeper/bookkeeper-4.4.0/bookkeeper-server-4.4.0-bin.tar.gz \
-&& tar xvzf  bookkeeper-server-4.4.0-bin.tar.gz \
-&& mkdir -p /opt/bk_all \
-&& mv bookkeeper-server-4.4.0/ /opt/bk_all/
-
-RUN wget http://www.apache.org/dist/zookeeper/zookeeper-3.5.1-alpha/zookeeper-3.5.1-alpha.tar.gz \
-&& tar xvzf  zookeeper-3.5.1-alpha.tar.gz \
-&& mkdir -p /opt/zk \
-&& mv zookeeper-3.5.1-alpha/ /opt/zk/
+ENV BOOKKEEPER_VERSION=4.5.0 \
```

Comment:
Is there a way to pull this from our gradle.properties? that way they stay in sync?

Reply:
Not really, it _could_ be made an argument but then everyone would have to specify it when building or it would have to have a default value which doesn't really seem to solve the problem anyway.